infrastructure.md

```diff
@@ -38,48 +40,87 @@ goes wrong with the bot.
 ([bot user account](https://github.com/rfcbot)) which helps manage async
 decision making on issues and PRs (typically RFCs). Team members can view any
-pending requests for review on [the FCP dashboard](http://rusty-dash.com/fcp).
+pending requests for review on [the FCP dashboard](https://rfcbot.rs/).
 
 Documentation on rfcbot commands can be found
-[in the dashboard repository](https://github.com/dikaiosune/rust-dashboard/blob/master/RFCBOT.md).
-
-Please contact [Adam Perry](https://github.com/dikaiosune) ("dikaiosune" on IRC)
-if something goes wrong with the bot, or open an issue on
-[the dashboard repository](https://github.com/dikaiosune/rust-dashboard/).
+[in the rfcbot repository](https://github.com/rust-lang/rfcbot-rs).
 
-## nagbot
+## rustbot
 
-[nagbot](https://github.com/aturon/nag-rs) is a bot that emails Rust team
-members to nag them for reviews. Please contact
-[Aaron Turon](https://github.com/aturon) ("aturon" on IRC) if something goes
-wrong with the bot, or open an issue on
-[its repo](https://github.com/aturon/nag-rs).
+[rustbot](https://github.com/rust-lang/triagebot) is a bot ([bot user
+account](https://github.com/rustbot)) to assist with managing issues and PRs
+to allow users to label and assign without GitHub permissions. See [the
+wiki](https://github.com/rust-lang/triagebot/wiki) for more information.
 
 ## DXR
 
 [DXR](https://dxr.mozilla.org/rust/source/) is a cross-referenced source index
 for Rust, allowing the Rust source tree to be navigated and searched with ease.
 It is generated by [rust-dxr](https://github.com/nrc/rust-dxr)
 
-## Others
-
-TODO: Describe these better.
-
-- [perf.rust-lang.org](http://perf.rust-lang.org/). Maintained by nrc.
-  [Source](https://github.com/nrc/rustc-perf).
-- [Rust Release Explorer](http://ashleygwilliams.github.io/rust-release-explorer/)
-- [rusty-dash.com](http://rusty-dash.com/).
-  [Source](https://github.com/dikaiosune/rust-dashboard).
-- [buildbot.rust-lang.org](http://buildbot.rust-lang.org/).
-  [Source](https://github.com/rust-lang/rust-buildbot). Maintained by acrichto.
-- [crater.rust-lang.org](https://crater.rust-lang.org/).
-  [Source](https://github.com/rust-lang-nursery/crater). Maintained by
-  tomprince.
-- [play.rust-lang.org](http://play.rust-lang.org/)
-  [Source](https://github.com/integer32llc/rust-playground). Maintained by
-  shepmaster.
-
-[rust-central-station]: https://github.com/alexcrichton/rust-central-station
+## perf
+
+[perf](https://github.com/rust-lang-nursery/rustc-perf) offers information
+about the performance of `rustc` over time.
+
+It is split into a data [collector] and a [web frontend]. The raw performance
+data is available [here](https://github.com/rust-lang-nursery/rustc-timing)
+and can be browsed on the [perf website](https://perf.rust-lang.org).
+
+[collector]: https://github.com/rust-lang-nursery/rustc-perf/tree/master/collector
+[web frontend]: https://github.com/rust-lang-nursery/rustc-perf/tree/master/site
```

Comment:
Could we add a quick blurb about the most common use case and commands to achieve that?

```
@bors try

<wait a bit; copy the $hash from the build>

@rust-timer build $hash
```

Reply:
I've never used the rust-timer bot, but I added a small blurb.  Ideally I would think documentation should be on the perf website (I searched around and couldn't find any anywhere, like who has permissions to use it, etc.).

Reply:
cc @Mark-Simulacrum